Tools/Scripts/webkitpy/benchmark_runner/run_benchmark.py

```diff
@@ -60,6 +60,7 @@ def config_argument_parser():
     parser.add_argument('--show-iteration-values', dest='show_iteration_values', action='store_true', help="Show the measured value for each iteration in addition to averages.")
     parser.add_argument('--generate-pgo-profiles', dest="generate_pgo_profiles", action='store_true', help="Collect LLVM profiles for PGO, and copy them to the diagnostics directory.")
     parser.add_argument('--profile', action='store_true', help="Collect profiling traces, and copy them to the diagnostic directory.")
+    parser.add_argument('--profiling-interval', default=None, help="Specify the profiling sampling rate.")
```

Comment:
Maybe mention in the description text the default interval, and examples of the time syntax you can give.